apkid/rules/dex/packers.yara

```diff
@@ -70,3 +70,25 @@ rule medusah_appsolid_dex : packer
   condition:
     is_dex and $loader and $main_activity
 }
+
+rule apkguard : packer
+{
+  meta:
+    description = "ApkGuard"
+    info = "http://apkguard.io/"
+
+  strings:
+    $method = "attachBaseContext"
+    $dexclassloader = "DexClassLoader"
+    $getclass = "getClass"
+    $getdeclaredmethod = "getDeclaredMethod"
+    $import = "java/lang/reflect/InvocationTargetException"
+    $getfilesdir = "getFilesDir"
+    $dex = "UEsDBB"
```

Comment:
This seems like the only thing unique to this packer. I think everything else would be really common for most packers. I'm not sure this is enough

Reply:
Hi Caleb,

I am going to try to obtain a couple of samples more. One with the same APK than before and a new one.